public_html/includes/formDefinitions/form_scheduler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+function defineList($s, $tableName) {
+	$l      = new listManagement($tableName);
```

Comment:
```php 
$l 
``` 
What does that mean, be descriptive enough naming your variables that someone can read it in a natural language to get an idea of what is going on with the code.